Source/SDK/DolittleClientService.cs

```diff
@@ -13,7 +13,7 @@ namespace Dolittle.SDK;
 /// <summary>
 /// Represents an implementation of <see cref="IHostedService"/> for the <see cref="IDolittleClient"/>.
 /// </summary>
-public class DolittleClientService : IHostedService
+public class DolittleClientService : IHostedService, IDisposable
```

Comment:
Is this usually the way to get this thing done? Does the hosting thing dispose of these services automatically, or is it done by the container?

Another way would be to just dispose of the client when it is stopped?

Reply:
We could do that as well, the IDisposable just ensures that the container disposes of it when the container is disposed to my knowledge